apps/frontend/src/app/1_atoms/ErrorRenderer/ErrorRenderer.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react';
+
+import { t } from 'i18next';
+
+import { Lottie } from '@sovryn/ui';
+
+import { translations } from '../../../locales/i18n';
+
+type ErrorRendererProps = {
+  error?: React.ReactNode;
+};
+
+export const ErrorRenderer: FC<ErrorRendererProps> = ({ error }) => {
+  return (
+    <div className="flex flex-col justify-center items-center flex-grow">
+      <div className="text-center whitespace-normal px-4">
+        <div className="w-full flex items-center justify-center">
+          <div className="w-64 h-64">
+            <Lottie animation="spaceScene" />
```

Comment:
Could you replace current animation with this one:
https://lottiefiles.com/69657-bug-hunting
and add the space one to `NotFoundPage` instead?

Once implemented we'll get it approved by Product, I really like them 😄